source/user-manual/reference/ossec-conf/vuln-detector.rst

```diff
@@ -225,6 +233,9 @@ Configuration block to specify vulnerability updates.
 |                    |                           |                    +--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 |                    |                           |                    | arch         | Parameterized link to download the feed file obtained from `Arch Security <https://security.archlinux.org>`_. You can find a guide on how to set it up :doc:`here<../../capabilities/vulnerability-detection/offline_update>`.                                            |
 |                    |                           |                    +--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                    |                           |                    | alas         | Parameterized link to download the feed file from Wazuh's own repository for `Amazon Linux <https://feed.wazuh.com/vulnerability-detector/ALAS/1/alas.json.gz>`_ or `Amazon Linux 2 <https://feed.wazuh.com/vulnerability-detector/ALAS/2/alas2.json.gz>`_.               |
```

Comment:
This has to be added to the `url` and `path` attributes for `os` instead of the independent options for the other providers.